patches/r6p2/series

```diff
@@ -17,3 +17,4 @@ r6p2/0016-mali-support-building-against-4.16.patch
 0018-mali-support-building-against-4.20.patch
 0019-mali-support-building-against-5.0.patch
 r6p2/0020-mali-support-building-against-4.17.patch
+r6p2/0021-Add-A64-support.patch
```

Comment:
And that way, it would apply to all the versions

Reply:
OK. I was not aware about that that. Thanks